stable/prometheus-operator/Chart.yaml

```diff
@@ -12,7 +12,7 @@ sources:
   - https://github.com/coreos/kube-prometheus
   - https://github.com/coreos/prometheus-operator
   - https://coreos.com/operators/prometheus
-version: 8.2.4
+version: 8.2.5
```

Comment:
Since it's a new feature, semver dictates that the minor version should be bumped. i.e. `8.3.0`

Reply:
Version bumped to 8.3.0